python-lib/sharepoint_client.py

```diff
@@ -204,9 +205,15 @@ def create_custom_field(self, list_title, field_title):
         self.assert_response_ok(response)
         return response
 
+    def valid_xml(self, to_format):
```

Comment:
Same comments here as for the sharepoint server review:
- rename the method to escape_xml
- add the case for the "/" character.